### DISCUSSION TITLE
Four Corners Custom Layout

### DISCUSSION THREAD

**Discussion by bryanculver** [ianyh/Amethyst#1634] [Show and tell]
Looking to make effective use of my large monitor, which can effectively be a 2x2 arrangement of 13" screens, I wanted a four-up display with each corner. I stumbled upon #1097 but didn't see anyone share a layout. So I wrote a custom layout to do just this: https://gist.github.com/bryanculver/4c181881cca81d9963f49a77dd5ac127

Yes I know it's verbose but it was much easier to debug and is pretty performant.

What it does:

- 1 window: entire screen
- 2 windows: even split left and right of screen
- 3 windows: first window is on left, 2nd and 3rd split vertically the right
- 4 windows: all four corners (as you'd expect)
- 5+ windows: window 1 is top left, window 2 is top right, window 3 is bottom left, windows 4+ vertically split the bottom right corner.

**Comment by QuinnB11** [comment]
This is awesome!
